pymc/printing.py

```diff
@@ -215,9 +217,20 @@ def _expand(x):
     names = [x.name for x in parents]
 
     if "latex" in formatting:
-        return r"f(" + ",~".join([r"\text{" + _latex_escape(n) + "}" for n in names]) + ")"
+        return (
+            r"f("
+            + ",~".join([_latex_text_format(_latex_escape(n.strip("$"))) for n in names])
+            + ")"
+        )
+    else:
+        return r"f(" + ", ".join([n.strip("$") for n in names]) + ")"
+
+
+def _latex_text_format(text: str) -> str:
```

Comment:
Okay for now, but I have a sense that this can be simplified since you're always using it on an output of `_latex_escape`

Reply:
I can add the call to `_latex_escape(text.strip("$")` inside `_latex_text_format` and call this also in the other 2 occurrences of `_latex_escape`. Then, we can also remove the naked `\text{` in there.